BenchBnb/BenchBnb/Models/Bench.cs

```diff
@@ -19,6 +19,12 @@ public class Bench
         [Required, Column("Longitude")]
         public float Longitude { get; set; } 
```

Comment:
`Required` attributes are redundant for non-nullable property types. Also, `Column` attributes are redundant unless you need to change the database table column name to something other than the property name.

Reply:
Fixed